plugnpy/parser.py

```diff
@@ -77,3 +99,7 @@ def exit(self, status=0, message=None):
     def set_copyright(self, copystr):
         """Set copyright string."""
         self._copyright = copystr
+
+    def _help_exit_unknown(self):
```

Comment:
this method is only used by a unit test - as its private its only intended for internal use in this class so is it needed?

Reply:
removed